app/controllers/homes_controller.rb

```diff
@@ -1,4 +1,8 @@
 class HomesController < ApplicationController
   def index
   end
+
+  def show
+  end
+
```

Comment:
Since this is never being used in `routes.rb` lets remove 4-7

Reply:
deleted it